lms/models/event.py

```diff
@@ -16,6 +16,7 @@ class Type(str, Enum):
         AUDIT_TRAIL = "audit"
         EDITED_ASSIGNMENT = "edited_assignment"
         SUBMISSION = "submission"
+        GRADE = "grade"
```

Comment:
Sorry @jon-betts forgot about this follow up when looking at the report PRs.

I can write those PRs if this name for the event looks fine.